cmd/agent/dist/conf.d/agentprofiling.d/conf.yaml.example

```diff
@@ -0,0 +1,32 @@
+## This file is overwritten upon Agent upgrade.
+## To make modifications to the check configuration, copy this file
+## to `conf.yaml` and make your changes on that file.
+
+init_config: {}
+instances:
+  - 
+
+    ## @param memory_threshold - integer - required - default: "0"
+    ## Set to either a size (for example, 10MB) or provide value in bytes (for example, 10485760). 
+    ## When this check runs, it will check if the Core Agent's memory usage is above this threshold.
+    ## If the threshold is exceeded, the check will generate a flare with memory and CPU profiles.
+    #
+    # memory_threshold: "0"
```

Comment:
For the default value of 0, I'm assuming that means the check doesn't happen? Please add a mention of that.